doi-transformer/src/test/java/no/unit/nva/doi/transformer/utils/HttpResponseStatus404.java

```diff
@@ -0,0 +1,20 @@
+package no.unit.nva.doi.transformer.utils;
+
+import org.apache.http.HttpStatus;
+
+public class HttpResponseStatus404<S> extends AbstractHttpResponse<S> {
+
+    public HttpResponseStatus404(S responseBody) {
+        super(responseBody);
+    }
+
+    @Override
+    public int statusCode() {
+        return HttpStatus.SC_NOT_FOUND;
```

Comment:
use HttpUrlConnection which is native to avoid dependency to the apache package.

Reply:
Using HttpUrlConnection...